src/Subsystems/Arm.h

```diff
@@ -4,6 +4,7 @@
 #include "WPILib.h"
 #include "ctre/Phoenix.h"
 #include "AnalogInput.h"
+#include <Preferences.h>
```

Comment:
Remove `#include <Preferences.h>`. 

`Preferences.h` is not needed in the header file. `Preferences.h` is needed in the source.